roles/openshift_metrics/tasks/install_hosa.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Generate Hawkular Agent (HOSA) Cluster Role
+  template: src=hawkular_openshift_agent_role.j2 dest={{mktemp.stdout}}/templates/metrics-hawkular-openshift-agent-role.yaml 
```

Comment:
can we follow the pattern you have in some of the other template actions where you do the following for increased readability?
```yaml
- name: 
  template:
    src: 
    dest:
```